packages/app-store/_pages/setup/index.tsx

```diff
@@ -16,6 +16,7 @@ export const AppSetupMap = {
   sendgrid: dynamic(() => import("../../sendgrid/pages/setup")),
   stripe: dynamic(() => import("../../stripepayment/pages/setup")),
   paypal: dynamic(() => import("../../paypal/pages/setup")),
+  "pipedrive-crm": dynamic(() => import("../../pipedrive-crm/pages/setup")),
```

Comment:
let's keep this key same as other keys.
```suggestion
  pipedrive-crm: dynamic(() => import("../../pipedrive-crm/pages/setup")),
```

Reply:
This change needs quotes becase of the dash "-" in its key, as noted by the other keys with dashes as well.

Reply:
yea got it